src/internal/Subscriber.ts

```diff
@@ -186,6 +192,18 @@ function defaultErrorHandler(err: any) {
   reportUnhandledError(err);
 }
 
+/**
+ * A handler for notifications that cannot be sent to a stopped subscriber.
+ * @param notification The notification being sent
+ * @param subscriber The stopped subscriber
+ */
+function handleStoppedNotification(
+  notification: { kind: 'N'; value: any } | { kind: 'E'; error: any } | { kind: 'C' },
+  subscriber: Subscriber<any>
+) {
+  setTimeout(() => config.onStoppedNotification?.(notification, subscriber));
```

Comment:
We should probably check to see if `onStoppedNotification` is undefined before scheduling the setTimeout:

```ts
const { onStoppedNotification } = config;
onStoppedNotification && setTimeout(() => onStoppedNotification(notification, subscriber))
```